Shared/Extensions/TextBlock/HtmlTextBlockBehavior.cs

```diff
@@ -1,11 +1,19 @@
-﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS10_0_18362_0_OR_GREATER
 using System;
 using System.Xml.Linq;
 using Windows.UI.Text;
```

Comment:
Should Windows.UI.Text; be in the #if?

Reply:
No, we need that namespace for `FontStyle` and `FontWeight`. It is not accessible under the Microsoft one